src/components/ResetPassword/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import ImageFader from '../Common/ImageFader';
+import { defineMessages, useIntl } from 'react-intl';
+import LanguagePicker from '../Layout/LanguagePicker';
+import Button from '../Common/Button';
+import { Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import axios from 'axios';
+import { useRouter } from 'next/router';
+
+const messages = defineMessages({
+  resetpassword: 'Reset your password',
+  password: 'Password',
+  confirmpassword: 'Confirm Password',
+  validationpasswordrequired: 'Password required',
+  validationconfirmpasswordrequired: 'Confirm Password required',
```

Comment:
Can't this use the same message as the one for the passwords match check, since an empty string wouldn't match an entered password?